aws/dynamodb/dynamo.tf

```diff
@@ -15,7 +15,7 @@ resource "aws_dynamodb_table" "reliability_queue" {
   }
 
   point_in_time_recovery {
-    enabled = true
+    enabled = var.env == "local" ? false : true
```

Comment:
Super minor comment - you could tighten this up like so:
```hcl
enabled = var.env != "local"
```

Reply:
Thanks for the catch Pat . Will fix